ops/delivery-e2e.cloudbuild.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+steps:
+  # Clean up any old Emblem instance cruft.
+  - name: '${_REGION}-docker.pkg.dev/${_PROJECT}/e2e-testing/deployer:${_E2E_DEPLOYER_SHA}'
+    entrypoint: /bin/bash
+    args:
+      - ./terraform/cleanup.sh
+    env:
+      - "PROJECT_NUMBER=$PROJECT_NUMBER"
+
+      # Project IDs
+      - "PROD_PROJECT=${_PROD_PROJECT}"
+      - "STAGE_PROJECT=${_STAGE_PROJECT}"
+      - "OPS_PROJECT=${_OPS_PROJECT}"
+
+  # Deploy a new Emblem instance
+  - name: '${_REGION}-docker.pkg.dev/${_PROJECT}/e2e-testing/deployer:${_E2E_DEPLOYER_SHA}'
+    entrypoint: /bin/bash
+    args:
+      - ./setup.sh
+    env:
+      - "PROJECT_NUMBER=$PROJECT_NUMBER"
+
+      # Project IDs
+      - "PROD_PROJECT=${_PROD_PROJECT}"
+      - "STAGE_PROJECT=${_STAGE_PROJECT}"
+      - "OPS_PROJECT=${_OPS_PROJECT}"
+
+      # Step-skipping
+      - "SKIP_TRIGGERS=true"
+      - "SKIP_AUTH=true"
+      - "SKIP_SEEDING=true"
```

Comment:
I don't think we want to skip seeding, unless it's blocking tests. That we can see seeded content in the website is a sign we have things working end-to-end.

Reply:
The issue is more the `read` call; I'll update #495 accordingly.